plugins/catalog-react/src/components/EntityOwnerPicker/EntityOwnerPicker.tsx

```diff
@@ -92,6 +93,10 @@ export type EntityOwnerPickerProps = {
   mode?: 'owners-only' | 'all';
 };
 
+const Popper = React.memo(function Popper({ children }: PopperProps) {
+  return <div>{children as ReactNode}</div>;
+});
```

Comment:
`popperProps` are no longer being forwarded to the `div`, could that break things?

Also wondering whether it might be enough to define this component outside, i.e. is `React.memo` actually needed?

Reply:
I think you are right, it should be enough without memo, but let me double check. 
I've stopped forwarding the other props because they were all related to the Popper react component and unrelated to the returned div, so we were getting some warnings for that

Reply:
yep, works! Thx for spotting this